### PR TITLE
Externalize fields in load balancer

### PR DESCRIPTION
**Pull request by palanik1**
Co-authored by : Sayandeep Sen sayandes@in.ibm.com

Signed-off-by: Palani Kodeswaran <palankod@in.ibm.com>

### CODE REVIEW

File: load_balancer.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ import (
 type LoadBalancer struct {
 	UUID            string
 	Name            string
-	vips            map[interface{}]interface{}
```
Comment (vtolstov) [inline load_balancer.go:29]: minor suggestion may be VIPs ?

**Review by vtolstov — APPROVED** [review]

File: load_balancer.go (hunk 1 above)
Comment (sdsen) [reply load_balancer.go:29]: Fixed. Thanks @vtolstov

**Review by vtolstov — APPROVED** [review]
Thanks

**Comment by palanik1** [comment]
Thanks @vtolstov

**Pull request merged**